control-plane/subcommand/inject-connect/v2controllers.go

```diff
@@ -78,6 +82,50 @@ func (c *Command) configureV2Controllers(ctx context.Context, mgr manager.Manage
 	//}
 
 	// TODO: register webhooks
+	mgr.GetWebhookServer().CertDir = c.flagCertDir
+
+	mgr.GetWebhookServer().Register("/mutate",
```

Comment:
Nope! Though i was under the assumption that only one of them could exist. this would require us to have another route on the mutating webhook which in itself is not an issue. we can do that in a follow up PR though since ti should be very straightforward!